### PR TITLE
2573 reorder runtime releases draft

### PR DESCRIPTION
**Pull request by natalie-bernhard**
#### What was fixed?  (Issue # or description of fix)
Issue #2573 
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Review by Sreejith-Websphere — COMMENTED** [review]
@natalie-bernhard one small suggestion here is to try to make different commits one for issue fix and other for code formatting for easy to debug

File: src/main/content/_assets/js/builds.js [hunk 1]
```diff
@@ -91,1298 +89,1352 @@ function isStagingSite() {
...
+    "webProfile9",
+    "microProfile5",
+    "javaee8",
+    "webProfile8",
+    "microProfile4",
+    "microProfile3",
+    "kernel",
+    "openliberty",
+  ];
+  var newLocations = [];
```
Comment (Sreejith-Websphere) [inline src/main/content/_assets/js/builds.js]: please use  self-explanatory variable names otherwise lgtm

**Comment by natalie-bernhard** [comment]
Removed formatting from PR and fixed issues with => notation

**Review by Sreejith-Websphere — APPROVED** [review]
lgtm

**Pull request merged**